### PR TITLE
feat(cwts): add Choose what to sync to fx_desktop_v2

### LINKED ISSUE #3213: Upgrade fx_desktop_v2 context to use Choose what to sync
Discussed this with @shane-tomlinson, we can finally get fx_desktop_v2 context into Firefox and with that also bring "choose what to sync" along!

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #3213

### CODE REVIEW

**Comment by vladikoff** [comment]
Ref https://bugzilla.mozilla.org/show_bug.cgi?id=1218022

**Comment by vladikoff** [comment]
@shane-tomlinson this should be ready to go!

File: tests/functional/firstrun_sign_up.js [hunk 1]
```diff
@@ -59,6 +59,33 @@ define([
         })
 
         .then(FunctionalHelpers.testIsBrowserNotified(self, 'fxaccounts:can_link_account'))
+
+        // user should be transitioned to the choose what to Sync page
+        .findByCssSelector('#fxa-choose-what-to-sync-header')
+        .end()
+
+        .findByCssSelector('button[type=submit]')
+        .click()
```
Comment (shane-tomlinson) [inline tests/functional/firstrun_sign_up.js]: nit: indent the click.

File: tests/functional/firstrun_sign_up.js [hunk 2]
```diff
@@ -59,6 +59,33 @@ define([
         })
 
         .then(FunctionalHelpers.testIsBrowserNotified(self, 'fxaccounts:can_link_account'))
+
+        // user should be transitioned to the choose what to Sync page
+        .findByCssSelector('#fxa-choose-what-to-sync-header')
```
Comment (shane-tomlinson) [inline tests/functional/firstrun_sign_up.js]: Is Fx Desktop able to accept web CWTS data already? If so, great, if not, we'll have to disable CWTS for the firstrun flow, and when the Fx patch lands, enable CWTS for firstrun, maybe using a different context?

File: app/scripts/models/auth_brokers/first-run.js [hunk 3]
```diff
@@ -23,6 +24,11 @@ define([
       VERIFICATION_COMPLETE: 'verification_complete'
     },
 
+    defaultCapabilities: _.extend({}, proto.defaultCapabilities, {
+      chooseWhatToSyncCheckbox: true,
+      chooseWhatToSyncWebV1: false
+    }),
```
Comment (vladikoff) [inline app/scripts/models/auth_brokers/first-run.js]: From @shane-tomlinson feedback, we disabled CWTS in first run. 

Also this branch has the old version once we are ready to enable CWTS in first run https://github.com/mozilla/fxa-content-server/tree/cwts-firstrun-future

**Comment by vladikoff** [comment]
I addressed the feedback ( https://github.com/mozilla/fxa-content-server/pull/3214#discussion_r43286298 ) and merging this now to test sync using latest.dev.lcip.org

Shane feel free to recommend tweaks and I'll add them in future PRs

**Pull request merged**